component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/record/RecordImpl.java

```diff
@@ -128,12 +128,12 @@ public BuilderImpl(final Schema providedSchema) {
             return entry;
         }
 
-        private Schema.Entry findOrBuildEntry(final String name, final Schema.Type type, final boolean nullable) {
+        private Schema.Entry findOrBuildEntry(final String name, final Schema.Type type) {
```

Comment:
I think that we must keep this schema parameter...

Reply:
`component-kitap` module has failing tests coming from your changes, see travis PR build.

Reply:
that's a a private method, so no large consequence with signature change.
for component-kitap; i just have to change unit test.